src/data/sampledata.ts

```diff
@@ -1,3 +1,18 @@
+/**
+ * Represents information about an MCAP (Model, Configuration, and Analysis Protocol) file and its associated data.
+ * Each entry in the array corresponds to a specific event, containing metadata about files, content, and event details.
+ *
+ * @typedef {Object} MCAPFileInformationNew
+ * @property {string} id - A unique identifier for the event.
+ * @property {Array<{file_name: string, signed_url: string}>} mcap_files - A list of MCAP files related to the event. Each file is represented by its name and a signed URL to access the file.
```

Comment:
In our index.d.ts file we have a type called FileType. I think it would be easier to say that mcap_files is an Array<FileType>.